### PR TITLE
[BUG FIX] Remove a typo and copyedit content

### LINKED ISSUE #2108: ACL SETUSER : https://redis.io/commands/acl-setuser/ - text change request
https://redis.io/commands/acl-setuser/

ACL SETUSER virginia on allkeys +set

....Then another SETUSER call can modify the user rules:

ACL SETUSER virginia +get

"The above rule will not apply the new rule to the user virginia, so other than [SET], the user virginia will now be able to also use the [GET] command."

Shouldn't the "not" in that opening sentence be removed? 
The second SETUSER rule seems to apply that change in my local setup.

### PR DESCRIPTION
**Pull request by nermiller**
Fixes #2108 plus copyedits

### CODE REVIEW

File: commands/acl-setuser.md [hunk 1]
```diff
@@ -1,32 +1,32 @@
 Create an ACL user with the specified rules or modify the rules of an
-existing user. This is the main interface in order to manipulate Redis ACL
-users interactively: if the username does not exist, the command creates
-the username without any privilege, then reads from left to right all the
-rules provided as successive arguments, setting the user ACL rules as specified.
+existing user. 
 
+Manipulate Redis ACL users interactively. If the username does not exist, the command creates
+the username without any privilege, then reads from left to right all the
+[rules](#aCL-rules) provided as successive arguments, setting the user ACL rules as specified.
```
Comment (nermiller) [inline commands/acl-setuser.md:6]:
```suggestion
[rules](#acl-rules) provided as successive arguments, setting the user ACL rules as specified.
```

File: commands/acl-setuser.md [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
 For example:
 
     ACL SETUSER virginia on +GET allkeys (+SET ~app1*)
 
-This sets a user with two sets of permission, one defined on the user and one defined with a selector.
-The root user permissions only allows executing the get command, but can be executed on any keys.
-The selector then grants a secondary set of permissions: access to the `SET` command to be executed on any key that starts with "app1".
+This sets a user with two sets of permissions, one defined on the user and one defined with a selector.
+The root user permissions only allow executing the get command, but can be executed on any keys.
+The selector then grants a secondary set of permissions: access to the `SET` command to be executed on any key that starts with app1.
```
Comment (nermiller) [inline commands/acl-setuser.md:29]:
```suggestion
The selector then grants a secondary set of permissions: access to the `SET` command to be executed on any key that starts with `app1`.
```

File: commands/acl-setuser.md [hunk 3]
```diff
@@ -50,42 +50,42 @@ another string (without any space in between), like "+SET".
...
+* `allchannels`: Alias for `&*`, it allows the user to access all Pub/Sub channels.
+* `resetchannels`: Removes all channel patterns from the list of Pub/Sub channel patterns the user can access.
+* `+<command>`: Adds the command to the list of commands the user can call. Can be used with `|` for allowing subcommands (e.g "+config|get").
+* `+@<category>`: Adds all the commands in the specified category to the list of commands the user is able to execute. Example: `+@string` (adds all the string commands). For a list of categories, check the `ACL CAT` command.
+* `+<command>|first-arg`: Allows a specific first argument of an otherwise disabled command. It is only supported on commands with no sub-commands, and is not allowed as negative form like -SELECT|1, only additive starting with "+". This feature is deprecated and may be removed in the future.
+* `allcommands`: Alias of `+@all`. Adds all the commands there are in the server, including *future commands* loaded via module, to be executed by this user.
+* `-<command>`: Remove the command to the list of commands the user can call. Starting Redis 7.0, it can be used with `|` for blocking subcommands (e.g., "-config|set").
```
Comment (itamarhaber) [inline commands/acl-setuser.md:73]: Same should be applied to https://redis.io/docs/manual/security/acl/

File: commands/acl-setuser.md [hunk 4]
```diff
@@ -50,42 +50,42 @@ another string (without any space in between), like "+SET".
...
+* `nopass`: The user is set as a _no password_ user. It means that it will be possible to authenticate as such user with any password. By default, the `default` special user is set as "nopass". The `nopass` rule will also reset all the configured passwords for the user.
+* `>password`: Adds the specified clear text password as a hashed password in the list of the users passwords. Every user can have many active passwords, so that password rotation will be simpler. The specified password is not stored as clear text inside the server. Example: `>mypassword`.
+* `#<hashedpassword>`: Adds the specified hashed password to the list of user passwords. A Redis hashed password is hashed with SHA256 and translated into a hexadecimal string. Example: `#c3ab8ff13720e8ad9047dd39466b3c8974e592c2fa383d4a3960714caef0c4f2`.
```
Comment (itamarhaber) [inline commands/acl-setuser.md:83]: Ditto

File: commands/acl-setuser.md (hunk 3 above)
Comment (nermiller) [reply commands/acl-setuser.md:70]: will do!

File: commands/acl-setuser.md (hunk 4 above)
Comment (nermiller) [reply commands/acl-setuser.md:80]: ok, I'll create another PR for that

**Pull request merged**